Data-Manager crashes on startup

data-manager crashes on startup: 

Steps to reproduce: 

1. Run `docker-compose up i40-aas-data-manager` 

You'll see in the logs:

```
i40-aas-data-manager_1             | > i40-data-manager@1.0.0 start /manager
i40-aas-data-manager_1             | > node ./dist/server.js
i40-aas-data-manager_1             | 
i40-aas-data-manager_1             | internal/modules/cjs/loader.js:797
i40-aas-data-manager_1             |     throw err;
i40-aas-data-manager_1             |     ^
i40-aas-data-manager_1             | 
i40-aas-data-manager_1             | Error: Cannot find module '../../src/config/swagger.json'
i40-aas-data-manager_1             | Require stack:
i40-aas-data-manager_1             | - /manager/dist/middleware/apiDocs.js
i40-aas-data-manager_1             | - /manager/dist/middleware/index.js
i40-aas-data-manager_1             | - /manager/dist/server.js
i40-aas-data-manager_1             |     at Function.Module._resolveFilename (internal/modules/cjs/loader.js:794:15)
i40-aas-data-manager_1             |     at Function.Module._load (internal/modules/cjs/loader.js:687:27)
i40-aas-data-manager_1             |     at Module.require (internal/modules/cjs/loader.js:849:19)
i40-aas-data-manager_1             |     at require (internal/modules/cjs/helpers.js:74:18)
i40-aas-data-manager_1             |     at Object.<anonymous> (/manager/dist/middleware/apiDocs.js:7:40)
i40-aas-data-manager_1             |     at Module._compile (internal/modules/cjs/loader.js:956:30)
i40-aas-data-manager_1             |     at Object.Module._extensions..js (internal/modules/cjs/loader.js:973:10)
i40-aas-data-manager_1             |     at Module.load (internal/modules/cjs/loader.js:812:32)
i40-aas-data-manager_1             |     at Function.Module._load (internal/modules/cjs/loader.js:724:14)
i40-aas-data-manager_1             |     at Module.require (internal/modules/cjs/loader.js:849:19) {
i40-aas-data-manager_1             |   code: 'MODULE_NOT_FOUND',
i40-aas-data-manager_1             |   requireStack: [
i40-aas-data-manager_1             |     '/manager/dist/middleware/apiDocs.js',
i40-aas-data-manager_1             |     '/manager/dist/middleware/index.js',
i40-aas-data-manager_1             |     '/manager/dist/server.js'
i40-aas-data-manager_1             |   ]
i40-aas-data-manager_1             | }
i40-aas-data-manager_1             | npm ERR! code ELIFECYCLE
i40-aas-data-manager_1             | npm ERR! errno 1
i40-aas-data-manager_1             | npm ERR! i40-data-manager@1.0.0 start: `node ./dist/server.js`
i40-aas-data-manager_1             | npm ERR! Exit status 1
i40-aas-data-manager_1             | npm ERR! 
i40-aas-data-manager_1             | npm ERR! Failed at the i40-data-manager@1.0.0 start script.
i40-aas-data-manager_1             | npm ERR! This is probably not a problem with npm. There is likely additional logging output above.

```

--- a/apps/data-manager/src/middleware/apiDocs.ts
+++ b/apps/data-manager/src/middleware/apiDocs.ts
@@ -1,6 +1,6 @@
 import { Router } from "express";
 import swaggerUi from "swagger-ui-express";
-import swaggerDocument from "../../src/config/swagger.json";
+import swaggerDocument from "../config/swagger.json";
 
 export const handleAPIDocs = (router: Router) =>
   router.use("/api-docs", swaggerUi.serve, swaggerUi.setup(swaggerDocument));

